src/tcpstream.c

```diff
@@ -156,6 +156,13 @@ void *run_ntttcp_sender_tcp_stream(void *ptr)
 					sockfds[i] = -1;
 					continue;
 				}
+				if (sc->tcp_nodelay && set_socket_tcp_nodelay(sockfd) == -1) {
+					ASPRINTF(&log, "cannot set option TCP_NODELAY for socket[%d]", sockfd);
```

Comment:
Include the error message returned by setsockopt in the error message. As returned by strerror(errno).

Here and two more occurrances below.